docs/client/introduction.mdx

```diff
@@ -3,21 +3,60 @@ title: Client SDKs
 
-Or, you can jump right in to the technical docs, by language, on the sidebar or below:
+-	The [**StatsigUser**](/client/concepts/user) object that you provide in the `initialize` call should include the user identifier, _userID_, that you use to identify the end-user of your application; the client SDK also generates a device identifier called _stableID_ to enable experiments where users aren’t signed in and a _userID_ is not available; you can choose to override this _stableID_ through **StatsigOptions** using the _overrideStableID_ parameter when you initialize the SDK
+
+:::info Using [**StatsigUser**](/client/concepts/user)
+
+Learn how to use `StatsigUser` before using a client SDK. 
+
+:::
+
+The configuration parameters received from Statsig servers in response to the `initialize` call persist until you either make another `initialize` call (recommended when the end-user starts a new session) or an `updateUser` call (recommended when an end-user attribute changes). These calls trigger a revaluation of all experiment configuration for the end-user.   
+
+### 2. Serving Variants
+
+ - After the `initialize` call completes, any subsequent `getExperiment` call synchronously returns the experiment configuration for the variant that your end-user is assigned to (you define these parameters when you set up an experiment on the Statsig console);  use the returned `config` object to `get` the value of the parameters required to implement the experiment variant
+ 
+ - If your experiment is part of a Layer, the `getLayer` call returns the experiment configuration for the variant that the your end-user is assigned to;  again, use the returned `config` object to `get` the value of the parameters required to implement the experiment variant
+  
+ - The `checkGate` calls returns _true_ if the end-user passes the feature gate rules that you've configured on the Statsig console, and _false_ otherwise 
+ 
+ - The `getConfig` call returns the dynamic configuration based on the rules that you've configured to target your end-users; use the returned `config` object to `get` the value of the parameters required to serve a dynamically configured app experience
+
+ Statsig recommends including a default value when you make the `get` call to fetch a parameter value from the returned `config` object. This will ensure that your application code falls back to the default value for a parameter that's not yet been configured in an experiment, layer, or dynamic config in the Statsig console. For convenience, Statsig by default returns _false_ when you check a gate that has not yet been configured.   
+
+### 3. Recording Exposure Checks & Custom Events
+
+-	The client SDK automatically flushes all accumulated exposure checks and logged custom events to the Statsig servers every 10 seconds; you can also force this flush by calling `shutdown` to push these accumulated exposure checks and custom events to Statsig servers 
```

Comment:
shutdown also resets the state of the SDK, so mention that they can call it on application exit or similar